### PR TITLE
kubo: Rename from 'go-ipfs'

### LINKED ISSUE #3776: [Request] go-ipfs renamed to kubo and released 0.14
New page is https://github.com/ipfs/kubo/releases

### PR DESCRIPTION
**Pull request by niheaven**
- Closes #3776

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Why not use GitHub releases and checkver?

**Comment by niheaven** [comment]
Faster and no blocks

**Review by rashil2000 — APPROVED** [review]

**Comment by hsn10** [comment]
doesn't work:

```
PS C:\WINDOWS\system32> scoop install -g kubo
Installing 'kubo' (0.14.0) [64bit]
kubo_v0.14.0_windows-amd64.zip (25,9 MB) [====================================================================] 100%
Checking hash of kubo_v0.14.0_windows-amd64.zip ... ok.
Extracting kubo_v0.14.0_windows-amd64.zip ... Could not find 'go-ipfs'! (error 16)
At C:\Program Files\scoop\apps\scoop\current\lib\core.ps1:613 char:9
+         throw "Could not find '$(fname $from)'! (error $($proc.ExitCo ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find 'go-ipfs'! (error 16):String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find 'go-ipfs'! (error 16)
```

File: bucket/kubo.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "64bit": {
+            "url": "https://dist.ipfs.io/kubo/v0.14.0/kubo_v0.14.0_windows-amd64.zip",
+            "hash": "sha512:6d2410bb5bd1012835e6bf7f12f079f459944a2de7b43440e066715f8f4cc9a5a9ac5ac89d5cf28209cef6828895397000013c45faadb630e75308e8918ac6bf"
+        },
+        "32bit": {
+            "url": "https://dist.ipfs.io/kubo/v0.14.0/kubo_v0.14.0_windows-386.zip",
+            "hash": "sha512:e882937cf4b491709993b207445758511428eae356fc7b665955a84b8cd4451e06d72ceb15db405ff431cec151d4e284de9d6de549895fdb64274d058db2403a"
+        }
+    },
+    "extract_dir": "go-ipfs",
```
Comment (redactedscribe) [inline bucket/kubo.json:16]: Should be:

```json
"extract_dir": "kubo",
```

Comment (rashil2000) [reply bucket/kubo.json:16]: Updated

**Pull request merged**